File: Selenium/Activities/activity7.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:

    driver.get("https://training-support.net/webelements/dynamic-controls")

    print("Page title is: ", driver.title)

    ip = driver.find_element(By.ID, "textInput")
    print("Is text field enabled : ",ip.is_enabled())

    driver.find_element(By.XPATH, "//button[text()='Enable Input']").click()
    print("Is text field enabled : ",ip.is_enabled())

    driver.quit()

File: Selenium/Activities/Activity7.java
package Example;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity7 {
	
	public static void main(String[] args) {
		  
		   WebDriver driver = new FirefoxDriver();
				    
		   driver.get("https://training-support.net/webelements/dynamic-controls");
				   
		   System.out.println("Title is: "+driver.getTitle());
		   
		   WebElement ip = driver.findElement(By.id("textInput"));
		   
		   System.out.println("Is text field enabled : "+ip.isEnabled());
		   
		   driver.findElement(By.xpath("//button[text()='Enable Input']")).click();
		   
		   System.out.println("Is text field enabled : "+ip.isEnabled());
		   
		   driver.quit();
}
}
